### DISCUSSION TITLE
Tiling window management in Athens

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#953] [General]
https://en.wikipedia.org/wiki/Tiling_window_manager

speculating the utility of window tiling in Athens

Examples of window tiling:
- blender (https://www.youtube.com/watch?v=4q6mLdO1E4Q)
- obsidian (https://help.obsidian.md/Panes/Pane+layout)

**Comment by jsmorabito** [comment]
>main benefits: tiling
main costs: UI dev time, user cognitive overhead
alternative: finish web app, so users can open Athens in multiple windows.
- Stuart

**Comment by solarkraft** [comment]
I find this less useful than tabs (could be combined, I suppose) or multiple windows (if you like tiling you can use a tiling window manager after all, I prefer stacking).

**Reply by jsmorabito** [reply]
is "stacking" referring to tabs? or another form of layout organization?

**Reply by solarkraft** [reply]
Stacking is a form of window management in which windows can be freely moved and overlap. Also referred to as "normal" ;)
After thinking about it a little I think internal tiling could be a more flexible/useful version of the side pane.

**Comment by yifanyin** [comment]
I'd like to bump this discussion about having multiple windows or multiple panes open. I guess this comes from the old habit of working with terminals and IDEs. But since the pages keep accumulating along with the Athens usability, it's hard to compare more than two pages at the same time with the sidebar (inherited from browser-based Roam). I really want more flexible note panes that native desktop app can achieve (somewhat like Obsidian, I guess. That tmux flash-back...) and the ability to remember panes every time I restart Athens.

**Reply by jefftangx** [reply]
We've thought about "remembering panes" and we called it "Workspaces" #281. Can still implement persistence there, regardless if our left sidebar continues to stay as basic as it is now.

Another partial solution to multiple panes is having multiple pages side-by-side, Andy Matuschak style. Or even multiple pages below each other on the y-axis. If we support both in an infinite canvas kind of way.... there are some Roam CSS themes that support this. But what should the default be? How to balance power vs simplicity?

Tmux all the way!
